### PR TITLE
Allows to read ingress and egress counter separately

### PR DESCRIPTION
**Pull request by daniele-moro**
Depends on: https://gerrit.onosproject.org/c/onos/+/25447

This PR consider only the read path. Code partially taken from:
- https://github.com/stratum/fabric-tna/pull/519
- https://github.com/stratum/fabric-tna/pull/517

### CODE REVIEW

File: src/main/java/org/stratumproject/fabric/tna/behaviour/upf/FabricUpfProgrammable.java [hunk 1]
```diff
@@ -645,48 +690,38 @@ public long tableSize(UpfEntityType entityType) throws UpfProgrammableException
...
-            if (counterCell.cellId().counterId().equals(FABRIC_INGRESS_UPF_TERMINATIONS_COUNTER)) {
-                stats.setIngress(counterCell.data().packets(), counterCell.data().bytes());
-            } else if (counterCell.cellId().counterId().equals(FABRIC_EGRESS_UPF_TERMINATIONS_COUNTER)) {
-                stats.setEgress(counterCell.data().packets(), counterCell.data().bytes());
-            } else {
-                log.warn("Unrecognized counter ID {}, skipping", counterCell);
-            }
-        });
-        return stats.build();
+        return processCounterEntryResponse(counterEntryResponse, Lists.newArrayList((long) cellId), type)
```
Comment (pierventre) [inline src/main/java/org/stratumproject/fabric/tna/behaviour/upf/FabricUpfProgrammable.java:723]: should we handle out-of-bound case ? READ not successful ?

Comment (pierventre) [reply src/main/java/org/stratumproject/fabric/tna/behaviour/upf/FabricUpfProgrammable.java:723]: you are doing `get(0)` what about if the list is empty. Sorry, I meant this.

Comment (daniele-moro) [reply src/main/java/org/stratumproject/fabric/tna/behaviour/upf/FabricUpfProgrammable.java:723]: The list can never be empty, it's initialized at the beginning of the `processCounterEntryResponse` method. Do you suggest to add a check before the return anyway?

**Comment by daniele-moro** [comment]
retest this please

**Pull request merged**